charts/traction/templates/_helpers.tpl

```diff
@@ -87,6 +88,33 @@ Return true if a database secret should be created
 {{- end -}}
 {{- end -}}
 
+{{/*
+Return true if a api secret should be created
+*/}}
+{{- define "acapy.api.createSecret" -}}
+{{- if not .Values.acapy.secret.adminApiKey.existingSecret -}}
+{{- true -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Return true if a api secret should be created
```

Comment:
Nit.: this is creating the wallet secret

Reply:
has been addressed in the following commit